Specification.md

```diff
@@ -1217,6 +1217,66 @@ Finally, make the appropriate changes to the local database (based on what actio
 
 ### Gossip Protocol
 
+#### Gossip Protocol Background
+
+Gossip Protocols ensure consistency in distributed systems with limited interactions. We define two mechanisms for the
+Public Key Directory: One active, one passive.
+
+#### Active Gossip (Checkpoints)
+
+The [Checkpoint](#checkpoint) Protocol Message type defined above (which allows one PKD to commit its current history
+onto another PKD's log) is one type of Gossip. With Checkpoints, the sender informs the recipient of their current
+Merkle root, and it is logged in the secondary ledger.
+
+If both Public Key Directory instances are independently operated, this allows third parties to verify the authenticity
+of the PKD that sent the Checkpoint.
+
+This inter-PKD mechanism is a **push** from the Sender to the Recipient. It requires an explicit action on the Sender to
+perform. This also requires the Recipient grant the Sender permission to publish a Checkpoint message.
+
+For completeness, we specify a permissionless, **pull**-based Gossip mechanism for replicating history from one PKD to 
+another. Care is taken to maximize the availability of data without preventing 
+[_Right To Be Forgotten_ takedowns](#message-attribute-shreddability).
+
+#### Passive Gossip (Mirroring)
+
+Third-party software and other Public Key Directory instances can be configured to stream new Protocol Messages from the
```

Comment:
What does it mean that only "new" messages will be stored by the Mirror?
So the Mirror gets the Root of the PKDs Sigsum and with that creates a *sub* copy of the Merkle Tree and the auxiliary database?
Or do we have an out of band copy of the whole data (tree +db) and from point x copy all new messages?

Maybe my question is "What does it mean to be a Mirror?", since Mirror implies to me a full copy.

Reply:
> What does it mean that only "new" messages will be stored by the Mirror?

See #19 which has a feed of historical messages. That will be used for replication.